cours/openstack/openstack-presentation.fr.md

```diff
@@ -18,14 +18,14 @@
 
 ### Mission
 
-"To produce an ubiquitous Open Source Cloud Computing platform that is
+```To produce an ubiquitous Open Source Cloud Computing platform that is
```

Comment:
Block \```code``` shouldn't be used here. It displays the text on a single line. `"` Should be used instead imo.

Reply:
I agree.